src/Sentry.Unity/Sentry.Unity.csproj

```diff
@@ -15,9 +15,9 @@
     </ItemGroup>
     <Delete Files="@(FilesToDelete)" ContinueOnError="true" />
   </Target>
-  
+
   <ItemGroup>
-    <ProjectReference Include="..\sentry-dotnet\src\Sentry\Sentry.csproj" />
+    <ProjectReference Include="..\..\modules\sentry-dotnet\src\Sentry\Sentry.csproj" />
```

Comment:
The slashes are killing me. It's funny how Rider just doesn't care and mixes them up.

Reply:
msbuild that supports either way. It was necessary to go from Windows to x-plat